src/layouts/Header/components/Navigator/Navigator.tsx

```diff
@@ -57,7 +67,8 @@ function Navigator({ currentUser }: NavigatorProps) {
   }
 
   // Fallback instead of MyContextSwitcher if not logged in
-  if (!currentUser) {
+  // If the owner doesn't exist, don't show anything
+  if (!currentUser && ownerData) {
```

Comment:
If the user is viewing an owner that doesn't exist, it just rendered the Avatar icon with a question mark, so just removing that entirely, but leaving in the user controls on the right hand side.